src/components/OneTimeCodeInput/index.tsx

```diff
@@ -137,6 +137,7 @@ export const OneTimeCodeInput: React.FC<OneTimeCodeInputProps> = ({
             value={value}
             key={index}
             label={label}
+            pattern="\d*"
```

Comment:
What is the difference between this and `type="number"`?

Reply:
Type number changes the way how the input is behaving in the tests, which caused a lot of problems 😢 Couldn't figure it out, so I wanted to look into that later, while this is having the same result.